docs/config/index.rst

```diff
@@ -220,22 +220,27 @@ To see what configuration parameters are defined for a given ``conf``::
      'auto_max_age',
      ...,
      'ietf_leap_second_auto_url']
-    >>> conf.auto_max_age
-    30.0
 
-You can also iterate through ``conf`` in a dictionary-like fashion::
```

Comment:
Maybe we still want to advertise that one can iterate through like `dict`. I vaguely remember having to do such a thing a long time ago for other reason, so I think this is still good to document even if you don't need it for description anymore.